tests/e2e/dynamic_provisioning.go

```diff
@@ -30,19 +31,28 @@ import (
 
 	awscloud "github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud"
 	ebscsidriver "github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 )
 
 var _ = Describe("[ebs-csi-e2e] [single-az] Dynamic Provisioning", func() {
```

Comment:
Could you create the snapshot tests in a separate `Describe` like `Describe("[ebs-csi-e2e] [single-az] Snapshot")`. So that we could add more snapshot related tests there (eg. snapshot with pre-provisioned, nvme, block volume)  if needed without mixing too much in in existing tests.

Reply:
Moved it.